vars/main.yml

```diff
@@ -230,10 +230,10 @@ manala_apt_keys_patterns:
     id:  86E50310
   sury_php_debian:
     url: https://packages.sury.org/php/apt.gpg
-    id: 4A7A714D
+    id:  4A7A714D
   sury_php_ubuntu:
     keyserver: hkp://keyserver.ubuntu.com:80
-    id: E5267A6C
+    id:        E5267A6C
```

Comment:
It's aligned with the content of the upper key, wich could be either "url" or "keyserver".
Or do i miss something ?